src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -868,6 +868,12 @@ private ImmutableArray<int> DecodeInterpolatedStringHandlerArgumentAttribute()
             {
                 return default;
             }
+            else if (ContainingType is PENamedTypeSymbol { IsExtension: true } containingPE
+                && containingPE.GetMarkerMethodSymbol() is MethodSymbol markerMethod
+                && (object)markerMethod == ContainingSymbol)
```

Comment:
>ContainingSymbol [](http://example.com/codeflow?start=43&length=16)

As an optimization consider starting with checking that ```ContainingSymbol``` is a method with a ```WellKnownMemberNames.ExtensionMarkerMethodName``` name.